src/main/java/com/minecolonies/coremod/colony/buildings/modules/LivingBuildingModule.java

```diff
@@ -385,21 +362,15 @@ private void inheritLastName(@NotNull final ICitizenData child, final String par
         child.setName(combinedName.toString().trim());
     }
 
-    @Override
-    public int getMaxInhabitants()
-    {
-        return getBuildingLevel();
-    }
-
     /**
      * Looks for a homeless citizen to add to the current building Calls. {@link #assignCitizen(ICitizenData)}
      */
     private void addHomelessCitizens()
```

Comment:
Should this be part of LivingBuildingModule, or separate? Do you expect LivingBuildingModule to be used for towers and training facilities? If so, this then creates the potential to mix the two assignment types.

Reply:
These places could override this module and create a SpecializedLivingBuildingModule that they will add for example.

Reply:
Ok, while that sends us partway down the inheritance path again, it's small enough that seems reasonable.